core/jobs/batch_jobs/suggestion_migration_jobs.py

```diff
@@ -0,0 +1,350 @@
+                            suggestion_models.STATUS_IN_REVIEW
+                        )
+                    )
+            )
+        )
+
+        explorations = (
+            self.pipeline
+            | 'Get all exploration models' >> ndb_io.GetModels(
+                exp_models.ExplorationModel.get_all(
+                    include_deleted=False))
+            | 'Fetch old to new content id mapping' >> beam.Map(
+                self._get_old_content_id_to_new_content_id_mapping)
+        )
+        exp_id_to_content_id_mapping = beam.pvalue.AsDict(explorations)
+
+        migrated_suggestion_results = (
+            unmigrated_translation_suggestion_models
+            | 'Transform to suggestion domain object' >> beam.Map(
+                suggestion_services.get_suggestion_from_model)
+            | 'Transform and migrate model' >> beam.Map(
+                self._update_content_id_in_translation_suggestion,
+                exp_id_to_content_id_mapping=exp_id_to_content_id_mapping)
+        )
+        migrated_suggestions = (
+            migrated_suggestion_results
+            | 'Filter oks' >> beam.Filter(
+                lambda result_item: result_item.is_ok())
+            | 'Unwrap ok' >> beam.Map(
+                lambda result_item: result_item.unwrap())
+        )
+        migrated_suggestion_job_run_results = (
+            migrated_suggestion_results
+            | 'Generate results for migration' >> (
+                job_result_transforms.ResultsToJobRunResults(
+                    'SUGGESTION PROCESSED'))
+        )
+
+        unmigrated_translation_suggestion_models_with_ids = (
+            unmigrated_translation_suggestion_models
+            | 'Add keys to unmigrated models' >> beam.WithKeys( # pylint: disable=no-value-for-parameter
+                lambda suggestion_model: suggestion_model.id)
+        )
+
+        suggestion_objects_list = (
+            {
+                'suggestion_model': (
+                    unmigrated_translation_suggestion_models_with_ids
+                ),
+                'suggestion': migrated_suggestions,
+            }
+            | 'Merge objects' >> beam.CoGroupByKey()
+            | 'Get rid of ID' >> beam.Values()  # pylint: disable=no-value-for-parameter
+            | 'Reorganize the suggestion objects' >> beam.Map(lambda objects: {
+                'suggestion_model': objects['suggestion_model'][0],
+                'suggestion': objects['suggestion'][0]
+            })
+        )
+
+        suggestion_objects_list_job_run_results = (
+            suggestion_objects_list
+            | 'Transform suggestion objects into job run results' >> (
+                job_result_transforms.CountObjectsToJobRunResult(
+                    'SUGGESTION MIGRATED'))
+        )
+
+        suggestion_models_to_put = (
+            suggestion_objects_list
+            | 'Generate suggestion models to put' >> beam.FlatMap(
```

Comment:
```suggestion
            | 'Generate suggestion models to put' >> beam.Map(
```

and modify _update_suggestion to return the model directly.

Reply:
Done!